src/include/esp-idf/bindings.h

```diff
@@ -157,4 +157,8 @@
 #include "host/util/util.h"
 #endif
 
+#include "mdns.h"
+#include "lwip/apps/netbiosns.h"
```

Comment:
Ditto - please check if this is a separate component, although I doubt that particular one.